src/assets/i18n/ubs/ua.json

```diff
@@ -16,7 +16,7 @@
     "max-length-comment": "Дозволено вводити не більше, ніж 255 символів",
     "minlength-short": "Поле містить недостатню кількість символів",
     "max-length": "Поле повинно містити 1-30 символів",
-    "pattern": "Дозволені тільки букви та спеціальні символи “-” (дефіс), “ ” (пропуск) та “'” (апостроф)",
+    "pattern": "Введіть коректний email",
```

Comment:
It's not quite clear what is meant by “valid”. 
Which pattern was meant to be used: google, i@ua, ukr@net ...

Reply:
Testers asked for this specific message as its used in user story